src/Microsoft.PowerShell.Commands.Management/commands/management/GetComputerInfoCommand.cs

```diff
@@ -558,13 +558,13 @@ private static MiscInfoGroup GetOtherInfo(CimSession session)
             // get secure-boot info
             //TODO: Local machine only? Check for that?
             FirmwareType fwType = FirmwareType.Unknown;
-            if (Native.GetFirmwareType(ref fwType))
+            if (GetFirmwareType(ref fwType))
```

Comment:
But that is what the code does now.  It never assigns a null value but always a FirmwareType value.  Are you saying we should change the behavior?  I feel the behavior is Ok as is, if "FirmwareType.Unknown" is fine to return if the firmware type cannot be found.